lib/plugins/plugin_api.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+# Plugin Class Extension for API Type Plugins
+class Plugin
+  private
+
+  def exec_api(data_from_chat, chat_text_array)
+    response = api_response(data_from_chat, chat_text_array)
+
+    if response.success?
+      response.body unless response.body.blank?
```

Comment:
As long as you are using Rails, why not use the positive form (that easier to read in my opinion) than the double negative?

    response.body if response.body.present?

See: https://github.com/rails/rails/blob/d66e7835bea9505f7003e5038aa19b6ea95ceea1/activesupport/lib/active_support/core_ext/object/blank.rb#L16